doc/changes.rst

```diff
@@ -7,7 +7,6 @@ Change Log
 
 *
 
-
```

Comment:
Alright, I think I changed it in the right place. Please, let me know if I have to change it again. Thanks.